scipy/special/tests/test_hyp2f1.py

```diff
@@ -1578,7 +1818,260 @@ def test_region5(self, hyp2f1_test_case):
         assert 1 < abs(z) < 1.1 and abs(1 - z) >= 0.9 and z.real >= 0
         assert_allclose(hyp2f1(a, b, c, z), expected, rtol=rtol)
 
-    # Marked as slow so it won't run by default.
```

Comment:
Seems this comment was removed unintentionally (as the following test still has `@pytest.mark.slow`)?

Reply:
Sorry. Strictly speaking I should have mentioned this in the PR message. I think this comment is no longer necessary. It started its life explaining why the test was marked slow even though the test was actually very fast. As test cases have been added, the test actually has become slow and the comment has been whittled down to just this. The shortened comment is actually not true. Slow tests do run by default, but on CI they only run for the full jobs.